utility/types/relay_chain.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import "github.com/pokt-network/pocket/shared/types"
+
+const (
+	RelayChainLength = 4
```

Comment:
Add comment explaining why this is what it is?

Reply:
Good point

Reply:
Assuming you added it, do resolve the thread.